Tools/Frame_params/SPARKKit-rover.param

```diff
@@ -0,0 +1,55 @@
+ACRO_TURN_RATE,90
```

Comment:
In #NOTE, please write a description indicating the model name and ArduRover version.
This is because automatic updates will fail if you load in a different version.

Reply:
@muramura Thank you for reply. I'll fix it 👍